service/committer/committer/committer.go

```diff
@@ -108,6 +129,20 @@ func (c *Committer) Run() {
 				continue
 			}
 
+			if types.IsPriorityOperationTx(poolTx.TxType) {
+				request, err := c.bc.PriorityRequestModel.GetPriorityRequestsByL2TxHash(poolTx.TxHash)
+				if err == nil {
+					priorityOperationMetric.WithLabelValues(strconv.FormatInt(request.L1BlockHeight, 10)).Set(float64(request.RequestId))
```

Comment:
Won't this cause too many metrics and eventually monitor get huge response?

Reply:
But I think this metric should records every changes of priorityRequest.requestID? 🤔